regression/get_coverage.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# Get a unique number to prevent collision of output files
+outputDir=`mktemp -d ./coverage_XXXXX`
+if [ $? -ne 0 ]; then
+    printf "ERROR: Could not create output directoy"
+    exit 1
+fi
+
+# Check that the previous command succeded, if not exit.
+commandStatus()
+{
+    if [ $? -ne 0 ]; then
+	printf "[ERROR]\n"
```

Comment:
This indentation seems a little strange and not consistent with the above if statement.